### ISSUE TITLE
TL-R860 is also not supported.

### ISSUE THREAD

**Issue by daiaji** [sta-c0000/tpconf_bin_xml#42]
Configuration file
[wrt2.bin.zip](https://github.com/sta-c0000/tpconf_bin_xml/files/13849614/wrt2.bin.zip)

**Comment by sta-c0000** [comment]
Right, not supported.  But thanks for the .bin file; it's a simple encrypted (zero padded) CRLF text file, not XML, with a 32 byte binary header (probably size and checksum).
I'm not going to work on this one, but decrypting the file does the job :)
`openssl enc -d -des-ecb -nopad -K 478DA50BF9E3D2CF -in wrt2.bin -out wrt2.txt -provider legacy # decrypt`
`tail -c +32 wrt2.txt # show config text file after skipping 32 bytes binary header`
